app/ante/eip712_test.go

```diff
@@ -587,10 +686,182 @@ func (suite *EIP712TestSuite) TestEIP712Tx() {
 			usdxToMintAmt:  90,
 			failCheckTx:    true,
 			errMsg:         "invalid pubkey",
-			updateTx: func(txBuilder client.TxBuilder, msgs []sdk.Msg) client.TxBuilder {
+			updateTx: func(_ client.TxBuilder, msgs []sdk.Msg) client.TxBuilder {
 				gasAmt := sdk.NewCoins(sdk.NewCoin("ukava", sdkmath.NewInt(20)))
 				return suite.createTestEIP712CosmosTxBuilder(
-					suite.testAddr2, suite.testPrivKey2, ChainID, uint64(sims.DefaultGenTxGas*10), gasAmt, msgs,
+					suite.testAddr2,
+					suite.testPrivKey2,
+					ChainID,
+					gasAmt,
+					msgs,
+				)
+			},
+		},
+		{
+			name:           "passes when domain matches expected fields",
```

Comment:
The added test cases here, the other changes are mostly just to resolve lint errors